test/tests.js

```diff
@@ -180,10 +180,10 @@ QUnit.test('undefined', function (assert) {
 
 QUnit.test('expires option as days from now', function (assert) {
 	assert.expect(1);
-	var expires = new Date();
-	expires.setDate(expires.getDate() + 1);
+	var days = 200;
+	var expires = new Date(new Date() * 1 + days * 864e+5);
```

Comment:
Is this dependent on the Date we're in? If so, can we create a new `Date` instance on a specific date? If it's too hard to do in JS, I'm not against introducing a third party devDependency such as [moment](https://www.npmjs.com/package/moment)

Reply:
`new Date()` gives us a timestamp representing milliseconds since January 1970 in UTC, which should be fine for the test and we won't need moment. 

If I revert the implementation to actually account for a DST change, I will need to adapt setting up the `expires` variable here to make use of code that would also account for DST (for instance `Date.prototype.setMilliSeconds()`, otherwise we will get 1 hour mismatches again occasionally. That said, what's important is that the test matches the implementation.

Reply:
I am wondering though - neither options seems more right or wrong to me. If we were, today at 16:00:00, writing a cookie with expires of 200 days..:

With accounting for DST, we'd get an expiration with the hour adapted 
`Mon, 29 Oct 2018 15:00:00 GMT`

Without accounting for DST, we'd get an expiration with the hour unchanged:
`Mon, 29 Oct 2018 16:00:00 GMT`

The latter seems even more expected to me, looking at it now.

Reply:
Given the fix for https://github.com/carhartl/jquery-cookie/issues/331 was more of an earlier attempt to fix the test (nobody asked that the hour when using days for expires would need to account for DST, and no behavior was specified either), and given the current implementation saved us a few bytes, I'm now rather going to suggest to keep the implementation as-is, and merge this PR, which fixes the test and makes it agnostic to the `Date` we're in (e.g. the test is being run at)...